pkg/agent/agent.go

```diff
@@ -71,8 +87,12 @@ func getPortName(req *cni.CNIRequest) string {
 }
 
 // invokes IPAM either internal or external and returns the CIDR
-func invokeIPAM(ipamType string) (string, error) {
+func invokeIPAM(ipamType string, fastPath bool) (string, error) {
 	if ipamType == InternalIPAM {
+		if fastPath {
+			ipAddrFast[3]++
+			return fmt.Sprintf("%s/24", ipAddrFast.String()), nil
```

Comment:
Don't assume ipv4

Reply:
this is just internal IPAM for testing purposes, we really want to use an external IPAM and there is a patch in progress under nimbess-cni to help with this. I'll add a comment.